<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="formula">
        <property name="text">
         <string>(1+2) / (3+4)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset theme="edit-delete">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GraphView" name="graph"/>
      <widget class="QSplitter" name="spltTrafoInputs">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lblShortTrafoDescr">
             <property name="text">
              <string>No transformation selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTrafo">
             <property name="text">
              <string>Transform</string>
             </property>
             <property name="icon">
              <iconset theme="emblem-system">
               <normaloff/>
              </iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="txtShortError">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="filter">
          <property name="placeholderText">
           <string>Search</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lstTransformations"/>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="txtLongError">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="placeholderText">
        <string>[No help topic open]</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLoadFormulas">
   <property name="text">
    <string>&amp;Load formulas...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QWidget</extends>
   <header>graph.graphview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
